requirements-manager.txt

```diff
@@ -1,2 +1,3 @@
+geoalchemy2
```

Comment:
Do we need this dependency?  Can we remove?

Reply:
Can you try removing so we can test what CI says (now that we have fully passing CI)?

Reply:
CI passes as a result.  Merging.  Thanks!

Reply:
@tomkralidis 
I removed the dependency, and it passed the CI.
However it depends on which requirements are installed for each test suite:
`requirements-provider.txt` also includes `psycopg2`
Should I remove also that dependency?

I think `requirements-manager.txt` should include **all and only** the requirements that need to be installed for the manager in addition to `requirements.txt`.
What is included into `requirements-provider.txt` is a different matter.

Or should we rather have a `requirements-postgresql.txt`, or `requirements-sqlalchemy.txt`?